Userland/Applications/TaskbarSettings/TaskbarSettingsWidget.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2018-2021, Filiph Sandström.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "TaskbarSettingsWidget.h"
+#include <AK/Assertions.h>
+#include <AK/JsonObject.h>
+#include <AK/QuickSort.h>
+#include <Applications/TaskbarSettings/TaskbarSettingsMainGML.h>
+#include <LibConfig/Client.h>
+#include <LibCore/DirIterator.h>
+#include <LibCore/File.h>
+#include <LibGUI/Application.h>
+#include <LibGUI/Button.h>
+#include <LibGUI/CheckBox.h>
+#include <LibGUI/FontPicker.h>
+#include <LibGUI/ItemListModel.h>
+#include <LibGUI/Label.h>
+#include <LibGUI/MessageBox.h>
+#include <LibGUI/OpacitySlider.h>
+#include <LibGUI/RadioButton.h>
+#include <LibGUI/SpinBox.h>
+#include <LibGUI/Widget.h>
+#include <LibGfx/Font.h>
+#include <LibGfx/FontDatabase.h>
+#include <LibKeyboard/CharacterMap.h>
+#include <LibVT/TerminalWidget.h>
+#include <spawn.h>
```

Comment:
A lot of these includes aren't used. I'm assuming you copied and pasted from a different settings app, but please try and only include what you actually use.